### PR TITLE
Cat

### PR DESCRIPTION
**Pull request by CatMO99**

### CODE REVIEW

File: _data/people.csv [hunk 1]
```diff
@@ -698,3 +698,8 @@ MORRIS,James Anthony,16.05.2002,,,"D3, M14"
 MORRIS,Enid Rosemary,12.10.2017,,,D3
 WEST,Mr & Mrs,,,,M20
 WHITFIELD,Phyllis,1963,,,"M21, E40"
+JONES,James Edward,August 1884,Age 13,of St Edwards Orphanage,B449
+STUART,Charlie,October 1885,Age 7,of St Edwards Orphanage,B450
+WITHEY,Mary Anne,June 22 1887,Age 55,Matron of St Edwards Orphanage,B451
+GROVE,Marianne,13 August 1877,Age 61,"daughter of the late Edward Grove Esq, of Shenstone Park Staffordshire",B452
+CARLO,Elizabeth,18 February 1887,Age 80,servant in Canon Pinder’s family for 49 years (see A7 and M9),B453
```
Comment (theoosborn) [inline _data/people.csv:705]: The ages here should just be the number, not with "Age"

**Review by theoosborn — CHANGES_REQUESTED** [review]

File: _data/people.csv (hunk 1 above)
Comment (CatMO99) [reply _data/people.csv]: OK have removed word 'age'.

File: _data/people.csv [hunk 2]
```diff
@@ -698,3 +698,145 @@ MORRIS,James Anthony,16.05.2002,,,"D3, M14"
...
+FARTHING,Henry Edward,1989,,,D27
+FARTHING,Gladys Mary,1990,,,D27
+ABBOTT,Brenda,2012,,,D28
+MAXFIELD,Margaret,17th August 2020,94,,D29
+STAVERT,Jane,1973,,daughter of Charles and Pamela Stavert,"D30, B438"
+STAVERT,Charles M (Wing Cdr),2008,,,D30
+STAVERT,Pamela W,2016,,,D30
+WEBSTER,Kathleen Isabel,1978,,,D31
+WEBSTER,Archibald Thomas,1988,,,D31
+PERKINS,Dudley Stewart,25.12.1973
```
Comment (theoosborn) [inline _data/people.csv:842]: This line with PERKINS needs 6 columns - it only has 3.

**Review by theoosborn — CHANGES_REQUESTED** [review]

File: _data/people.csv (hunk 2 above)
Comment (CatMO99) [reply _data/people.csv:842]: Yes I know I had to interrupt typing to get your sausage rolls!!!!

**Review by theoosborn — APPROVED** [review]

**Pull request merged**